src/components/ArtlcleCard.jsx

```diff
@@ -16,16 +17,25 @@ export default function ArticleCard() {
     });
   }, [article_id]);
 
+  const handleClick = (event) => {
+    event.target.disabled = true;
+    setVotesToAdd(1);
+    api.patchArticleVotes(article_id, 1);
+  };
   return (
     <div>
+      <br />
+      <Button variant="contained" color="primary" onClick={handleClick}>
+        Like This Article
+      </Button>
       <h2>Article</h2>
       <ul className="article-list">
         <li key={article.article_id}>
           <h3>{article.title}</h3>
-          <p>Written by: {article.owner}</p>
+          <p>Written by: {article.author}</p>
           <p>{article.article_body}</p>
```

Comment:
I think this should be article.body rather than article_body, I can't see the actual article text at the mo!